code/modules/vending/job_disk.dm

```diff
@@ -0,0 +1,39 @@
+//This one's from bay12
+/obj/machinery/vending/cart
+	name = "\improper PTech"
+	desc = "Job disks for PDAs."
+	product_slogans = "Disks to go!"
+	icon_state = "cart"
+	icon_deny = "cart-deny"
+	light_color = LIGHT_COLOR_WHITE
+	products = list(/obj/item/modular_computer/tablet/pda = 15,
+					/obj/item/computer_hardware/hard_drive/role/medical = 5,
+					/obj/item/computer_hardware/hard_drive/role/chemistry = 5,
+					/obj/item/computer_hardware/hard_drive/role/brig_physician = 3,
+					/obj/item/computer_hardware/hard_drive/role/security = 5,
+					/obj/item/computer_hardware/hard_drive/role/detective = 3,
+					/obj/item/computer_hardware/hard_drive/role/engineering = 5,
+					/obj/item/computer_hardware/hard_drive/role/atmos = 5,
+					/obj/item/computer_hardware/hard_drive/role/signal = 5,
+					/obj/item/computer_hardware/hard_drive/role/signal/toxins = 5,
```

Comment:
what is the difference between those two? Do you need both or is the toxin one enough?

Reply:
signal: only has signal program
toxins: signal + atmos scan + chem scan.
actually pathing is weird.

Reply:
remove normal signal and leave toxins I guess, you don't need them doubled